src/model/jwks.rs

```diff
@@ -164,15 +179,56 @@ impl Jwk {
     }
 }
 
+fn generate_x509_cert(key_pair: &PKey<Private>) -> Result<X509, ErrorStack> {
+    let mut x509_name = X509NameBuilder::new()?;
+    x509_name.append_entry_by_text("C", "IT")?;
+    x509_name.append_entry_by_text("O", "Prima CA")?;
+    x509_name.append_entry_by_text("CN", "Prima CA")?;
```

Comment:
This project is meant to be public and not just internal to prima. I think this should be either customizable, or changed to something more neutral, like "LocalAuth0"

Reply:
See this [comment](https://github.com/primait/localauth0/pull/25#discussion_r986921659)